packages/terra-data-grid/src/utils/WorklistDataGridUtils.js

```diff
@@ -2,7 +2,7 @@
  * The Object representation of the row selection column.
  */
 const ROW_SELECTION_COLUMN = {
-  id: 'WorklistDataGrid-rowSelectionColumn',
+  id: 'table-rowSelectionColumn',
```

Comment:
This line of code should not have been changed.

Reply:
Updated.